general/envsetup/envsetup.go

```diff
@@ -128,7 +132,11 @@ func (ftc *EnvSetupCommand) getNewServerDetails() (serverDetails *config.ServerD
 
 		// Wait for 'ready=true' response from MyJFrog
 		if resp.StatusCode == http.StatusOK {
-			ftc.progress.SetHeadlineMsg("Ready for your DevOps journey? Please hang on while we create your environment")
+			if !readyMessageDisplayed {
+				ftc.progress.ClearHeadlineMsg()
+				ftc.progress.SetHeadlineMsg("Almost done! Please hang on while JFrog CLI completes the setup")
```

Comment:
This message is a bit inaccurate, we are waiting here for a "ready" response from the website side...

Reply:
I agree that the message not super specific, but I wouldn't say it's incorrect. The user shouldn't be concerned with the MyJFrog integration details.